client/js/store/procedure/SegmentFilter.js

```diff
@@ -7,6 +7,8 @@
  * All rights reserved
  */
 
+import Vue from 'vue'
```

Comment:
I thnk this is not the propper way to fix that. Please try `this.$delete` instead